### ISSUE TITLE
Add brew service support

### ISSUE THREAD

**Issue by Zk2u** [tonarino/innernet#286]
Using innernet on macOS is currently very annoying - it doesn't update properly when new nodes are added and doesn't automatically start. 

We should support enabling a service in the brew tap so we can use `brew services start innernet@...`.

**Comment by mcginty** [comment]
Agreed, and I'd also like to get innernet in the official brew repository. macOS innernet needs a bit of love :).
